src/main/java/com/conveyal/taui/models/OpportunityDataset.java

```diff
@@ -0,0 +1,34 @@
+package com.conveyal.taui.models;
+
+public class OpportunityDataset extends Model {
+    /** The human-readable name of the data source from which this came */
+    public String sourceName;
```

Comment:
Worth adding examples of what we mean by "data source" (e.g. uploaded shapefile, or LODES via seamless census extract)